01_fib/brnbke/main_test.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote(`0
+1
+1
+2
+3
+5
+8
+13
+`)
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main(), expected %v, sadly got %v", expected, actual)
+	}
+}
+
+func TestFib(t *testing.T) {
```

Comment:
Does not completely check the output. 
Just verifies if the last value as fib function is correct or not. Previous values are not tested.

Reply:
Yep fair enough, have reworked the tests to do this.